dev_tool/src/local/otel.ts

```diff
@@ -0,0 +1,155 @@
+import LabeledProcessRunner from '../runner.js'
+import { checkDockerInstalledAndRunning } from '../deps.js'
+import { commandMustSucceedSync } from '../localProcess.js'
+import { command } from 'yargs'
+
+export async function runOtelLocally(runner: LabeledProcessRunner) {
+    await checkDockerInstalledAndRunning()
+    const psOut = commandMustSucceedSync('docker', [
+        'ps',
+        '-a',
+        '--format',
+        '{{ .Names }}\t{{ .Status }}',
+    ])
+
+    for (const line of psOut.split('\n')) {
+        const [name, status] = line.split('\t')
+        if (
+            name.startsWith('managed-care-review-otel-collector') &&
+            status.startsWith('Up')
+        ) {
+            console.log(
+                'ERROR: An instance of otel-collector is already running. In order to run `./dev local otel` you must first stop the instance.'
+            )
+        }
+        commandMustSucceedSync('docker', [
+            'compose',
+            'down',
+            '--remove-orphans',
+        ])
+    }
+
+    await runner.runCommandAndOutput(
+        'docker otel',
+        ['docker', 'compose', 'up'],
+
+        '.',
+
+        // this line is printed basically when things are ready.
+        // "ready to accept connections" is repeated so we can't use it.
+        { awaitFor: 'Everything is ready. Begin running and processing data.' }
+    )
+}
+
+// Attempt without docker compose. Network is failing between frontend and containers:
```

Comment:
I was having a hard time iterating with just `dev_local` as I was figuring this out, so I switched over to using `docker compose` while I was testing getting ports and exporting correct. Then I tried to bring those settings into `dev_local` without using `docker compose`, but I kept hitting network issues trying to get the otel collector talking to jaeger.

If we have a reason to not use `docker compose` with the docker config files we can try to go down the path below. 

If not, I'll delete this before merge.